### ISSUE TITLE
Any updates?

### ISSUE THREAD

**Issue by juanddp** [machinamentum/CTR64#9]
Is the project still alive? :/

**Comment by Eiskristall** [comment]
I hope this project is alive :p

**Comment by machinamentum** [comment]
Sorry about the lack of updates! I've been really busy with work and other projects, so this is going to be on a semi-permanent hold. I may revisit this at some point, though by that time I'm not sure if people would still be interested in 3DS homebrew since the Switch is the new kid on the block.

**Comment by loserVEVO** [comment]
please keep updating it, i dont want the only n64 emulator for the ds to be discontinued

**Comment by mrTruch** [comment]
Yes we are always interested, and expecting N64 mini, please reconsider.
